docs/remove-specified-tags-from-all-products/script.liquid

```diff
@@ -0,0 +1,84 @@
+{% assign tags_list = options.tags_to_remove__required_string | split: "," %}
```

Comment:
Hey @acflint! Can you refactor this to use the `array` option type? Specifying the option as `options.tags_to_remove__array_required` will allow the user to explicitly add tags one by one. Besides the UI upside, this means that you won't have to split the string, and there won't be any concerns about spaces on either side of a comma delimiter.

Reply:
Yes! Forgot about the array option type :) Thanks!